js/button.js

```diff
@@ -107,7 +107,7 @@
       var $btn = $(e.target)
       if (!$btn.hasClass('btn')) $btn = $btn.closest('.btn')
```

Comment:
Pretty sure your check needs to happen before this line; otherwise, `$btn` will *always* be the `label.btn`.